src/lib/swapper/utils.ts

```diff
@@ -97,17 +97,18 @@ export type MonadicSwapperAxiosService = ReturnType<typeof makeSwapperAxiosServi
 export const getEnabledSwappers = (
   { LifiSwap, ThorSwap, ZrxSwap, OneInch, Cowswap }: FeatureFlags,
   isCrossAccountTrade: boolean,
-) => {
-  return [
-    LifiSwap && SwapperName.LIFI,
-    ThorSwap && SwapperName.Thorchain,
-    ZrxSwap && SwapperName.Zrx,
-    OneInch && SwapperName.OneInch,
-    Cowswap && SwapperName.CowSwap,
-  ]
-    .filter(isTruthy)
-    .filter(swapperName => {
-      const swapperSupportsCrossAccountTrade = isCrossAccountTradeSupported(swapperName)
-      return !isCrossAccountTrade || swapperSupportsCrossAccountTrade
-    })
+): Record<SwapperName, boolean> => {
+  return {
+    [SwapperName.LIFI]:
+      LifiSwap && (!isCrossAccountTrade || isCrossAccountTradeSupported(SwapperName.LIFI)),
```

Comment:
Not a regression on this PR, though I think it's a recent regression - when we the user inputs a cross-account trade we used to show a "Cross-account trade now supported" or similar error message on the swapper, still showing the swapper. We no longer do (e.g. we'll never see 0x show up if another receive account is selected.

Reply:
Will tackle here https://github.com/shapeshift/web/issues/6075